index.html

```diff
@@ -2030,6 +2030,101 @@ <h4>Presentations Using Derived Credentials</h4>
             </figcaption>
           </figure>
         </section>
+        <section>
+          <h4>Presentations Including Holder Claims</h4>
+          <p>
+A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
+a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
+any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
+<a>verifiable credential</a> is the <a>holder</a>, the <a>claims</a> in that
+<a>verifiable credential</a> are considered to be <em>self-asserted</em>.
+Such self-asserted claims can be secured by the same mechanism that secures
+the <a>verifiable presentation</a> in which they are included or by any
+mechanism usable for other <a>verifiable credentials</a>.
+          </p>
+          <p>
+The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
+these <a>claims</a> can include statements about the <a>holder</a>, one of the
+other included <a>verifiable credentials</a>, or even the <a>verifiable
+presentation</a> in which the self-asserted <a>verifiable credential</a> is
+included. In each case, the <code>id</code> <a>property</a>
+is used to identify the specific <a>subject</a>, in the object where the
+<a>claims</a> about it are made, just as it is done in
+<a>verifiable credentials</a> that are not self-asserted.
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+All of the normative requirements defined for <a>verifiable credentials</a>
+apply to self-asserted <a>verifiable credentials</a>.
+          </p>
+          <p>
+When a self-asserted <a>verifiable credential</a> is secured using the same
+mechanism as the <a>verifiable presentation</a>, the value of the
+<code>issuer</code> <a>property</a> of the <a>verifiable credential</a>
+MUST be identical to the <code>holder</code> <a>property</a> of the
+<a>verifiable presentation</a>.
```

Comment:
Seems like we can perhaps remove that text from validation, if this section repeats it.

Reply:
I think that the language already present is simpler and is sufficient.
The holder is the entity constructing both the VP and the self-asserted VC, so they can make sure those values are identical, whether they choose to use an object with an `id` or just a URL.

Plus, we already have this line above, which was introduced specifically to avoid pointing back to each normative section of the spec that also applies: "All of the normative requirements defined for verifiable credentials apply to self-asserted verifiable credentials."

I would prefer to not make this change.

Reply:
I'm not going to block this PR if the change isn't made either.

Reply:
as an aside, presentations and credentials don't require `id`, but when one is present, the claims are bound to the `id`.

Reply:
I'm this suggestion should be applied, we can clean up when the dust settles.